src/biokbase/narrative/tests/test_appmanager.py

```diff
@@ -398,55 +379,16 @@ def test_run_app_batch_good_inputs(self, auth, c):
             tag=self.test_tag,
         )
         self.assertIsInstance(new_job, Job)
+        self.assertEqual(self.jm.get_job(self.test_job_id), new_job)
         self._verify_comm_success(c.return_value.send_comm_message, False)
 
-        job_meta = {
-            "batch_app": self.test_app_id,
-            "batch_size": len(params),
-            "batch_tag": self.test_tag,
-        }
-
-        expected = {
-            "app_id": BATCH_APP["APP_ID"],
-            "app_version": BATCH_APP["VERSION"],
-            "job_id": self.test_job_id,
-            "extra_data": job_meta,
-            "params": [
-                {
-                    "batch_params": [
-                        {
-                            "params": [self.expected_app_params],
-                            "source_ws_objects": [],
-                        }
-                        for _ in range(len(params))
-                    ],
-                    "meta": {
-                        **job_meta,
-                        "tag": BATCH_APP["TAG"],
-                        "token_id": TOKEN_ID,
-                    },
-                    "method_name": self.test_app_method_name,
-                    "module_name": self.test_app_module_name,
-                    "service_ver": self.test_app_version,
-                    "wsid": 12345,
-                }
-            ],
-            "tag": BATCH_APP["TAG"],
-        }
-
-        for attr in JOB_ATTRS:
-            if attr in expected:
-                self.assertEqual(getattr(new_job, attr), expected[attr])
-            else:
-                self.assertEqual(getattr(new_job, attr), JOB_ATTR_DEFAULTS[attr])
-
```

Comment:
Kind of just deleted this since `Job.from_job_id` would return mocked information anyway. This logic if applicable should go into the dry run test

Reply:
Yeah, the dry run test looks about as comprehensive as it can be. It tests the job runner inputs, which go to ee2, which make the state, which would be reflected in these attributes. I think we can just safely remove all this stuff